### PR TITLE
speed up filling background function by apply Homography

### PR DESCRIPTION
**Pull request by studentBC**

### CODE REVIEW

File: fillBackground.py [hunk 1]
```diff
@@ -0,0 +1,112 @@
+import cv2
+import numpy as np 
+
+from tqdm import tqdm
+from matcher import matcher
+
+class fillBackGround:
```
Comment (chiahaol) [inline fillBackground.py]: For class, we always use Pascal case. Change class name to BackgroundFilling
Also, file name should be snake case, ex: background_filling.py

File: fillBackground.py [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+        print('Go fill background ...')
+        n, height, width, channel = bg.shape
+        match = matcher()
+        sampleBG = []
+        # F1=H×F2
+        for i in tqdm(range(1, n, 10)):
+            H = match.match(bg[i-1], bg[i], 'right')
+            # print(H)
+            filled = False 
+            for j in range(height):
```
Comment (chiahaol) [inline fillBackground.py]: I think we could do matrix multiplication instead of for loop here

Comment (studentBC) [reply fillBackground.py]: ok, let us rely on your multiplication

File: fillBackground.py (hunk 1 above)
Comment (studentBC) [reply fillBackground.py]: done, Sir

**Review by laurenchen0631 — APPROVED** [review]

**Pull request merged**